src/utilities/axios/axios.js

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios';
+
+const instance = axios.create({
+  baseURL: import.meta.env.VITE_API_BASE_URL,
+  timeout: 1000,
```

Comment:
No big deal, but I would increase it to 2-3 sec

Reply:
My wife tells me the same thing every night..